packages/components/src/components/Dropdown/styles/dropdown.module.scss

```diff
@@ -53,6 +49,10 @@
         order: 2;
         margin-left: auto;
     }
+
+    &:not(&:hover) {
```

Comment:
If we leave the mouse on the item then switch to keyboard navigation, the outline isn't present.

https://github.com/user-attachments/assets/22981af1-baad-476c-b460-fbd5f8a72b77

Reply:
yes, i also noticed that, but i believe that its not a big issue.

The alternative would be to manually keep track of a separate selected state for the background and the outline. This would add quite a bit of hacky code which i believe is not worth it.

As there is no a11y guideline about focus indicators in select/dropdown elements this does not create an accessibility issue but is just a nice to have.